tf_quant_finance/examples/jupyter_notebooks/Forward_Backward_Diff.ipynb

```diff
@@ -0,0 +1,247 @@
pFQqcXZ2ZteuXWhoaLC1tcXS/Apa2pqoqqriH+NB/IkQhnuU5PxLJfUFQtwDHDG1MkLPWBvHIFNWVldoypMhaxjBP8oDUysjPAIdCYz1oDKzjZx/q8HOxRxHD2t8I1yxdTYn5w+11BeLCU/yxs7FnIhkb6py2ym404hvuAuu3raExHtQktZKeZaAmH2+BEd7YGCsS8G9VvR1TThyNgbvIBuq8iVUF4kxtFnHwV2Pro4Jmir6OHI6hr1Hg2iu6aY4u52EfT5EJ3pSlCOktaGXIy8F4+BkTsa9JlRU4OTZMMSiYYrzO0g54IuHlzUZqa1oaKpx+mwYJSVihO1DXL4cxcDAJIXFYo4dCURTU42sXCFR4a7Y2hiTlS/E19sGVycLsgtF6GguYWGmS12bgkPJQairq5FXLuHlYyHI+5Q0CQd475U4MkpELC6vcjDOm8xyMR5OFmhpqlPe0svtV5JIKxdhbm5OuK8zRc3dnE3wpUo8yOTUDA5G6tR1KYl3M6FSNoqWuip25sZUift4JSmIrKZOIj0cHhLxwgIG6tA5Mc/L8aEUiro4GuhJrlCOv701Z8N8iXK1R1fzz+NxMDk5SXt7O8bGxnh7e+/sxaqoqOz0Z83MzHY2J7aJZ3Z2luHhYXp7e59ZZvxd7Ym1tTU6OzvR09PbedLU0tLC0NBwp0+8LeyoqamhurqanJwcfvOb3zAyMkJiYuIT7/lpvHR//etfMzs7S35+PgYGBvzqV7/izJkzz/MW/7hJ97vsHR81YZ6fn0ckEjE5OYmXlxcODg5PvfD9tNlk34bNzU0GBgbo6OhAQ0MDLy8vTExMdh65+sTDfLD3v7Oxvol7kBNq6mroG+kSdzwY5922VGe2kvnP5UxPT7OmNYeNrTX7TsUREONOVVYrOf9ShYmlIa4+dphZGxF3LJCOhh4y/rkCLR1NPIOdcPK0xjvEmYIv6qnKFhCW7I21gymRe3dTnSek4G4TgdG7cPKwJjDGjcJ7TdQWiok96ItvqDNrq+tk/bEefQMdYvb64RPsRO6dJhQD80QmexCV6E1lfifNtV2Y2m/i4mFCQ3k/HYJ+9hzywWWXFRl3m9jcgoMnghgdniY3vY2IWHei4tzJSm1hbHSW0+fD6euboDBfxL79vtjZmZCVJcDa2oiUlN2kZ7axsrzG+XPh5OWLGB6Z5rXL0RSWSejuHSc51oaaxgHmF+HD6wfJKBAyN7/MuWPBpOYL0NbWICLQmZwyMX4eNmhqqFFcJ+PtM9EU1cuYml3iaMJusirFeDpZsLm5RX3HAO+cjOJuWTsudpYYGejSJFfwyaV9PKjqwN3ekqWVVWRD4+w206Sxf5JAS13qesfQVNlkc22V/ulFTkYFUNzRw6nQ3WQLZAQ72yIYVGCoo8WlyIA/C+Gura0hkUiYnJzE19cXc3Pzp1JSbkfsPLo5YW5ujqqqKgsLC4yOjtLb2/sVmfHT+j2MjY3tbEo4Ozs/cWAXGRnJl19+SU5ODjdu3MDLy+upMtyexkv3Zz/7Gbdu3cLBwQEvLy9u3LjBxx9//DxPHz9ua8fvwvYqi0gkQiQS4eTkREhIyDMPxNTV1Z/ZlnHbU7e2tpaVlRXCw8MxMjJicHCQsbExVldXUVVVRc9Il4B4L/70P7L5f9k78/io6nP/v2cyk8wkM5lM9n3f95Uk7EUWQUVqq+KG1uq1tlV/VtsiVYvVe7XWe3ut1boULW5Vel0RRFBAEMgKCYGQPSH7nsxkJrNklt8fJ3NIBDQsoqX9vF68Xnrm5MyZmXM+5/k+z+f5PP/vksep2lMnHmPW0gwefOvHJM+OYNvLB/jov8uRO5VIJBKS82N4avMvSCmI4bm1m3j5dx9gtzvw8Vez/tXbmb0ik9ee3Mqrv9+C0+kkvTCW9X/7MeMGMw/f9BL9nSP4h2h45JUf4any4NGfvEb38SGiEoJ44E/XM9ir4/f/7y0mrDau/9kiii5J5rU/fUbFF/XYpDqWrU6l9dgQtSUjpGUmcM9vV9Lboaf5kJkFlxSy+vZiGo708sZfP8UmHSCzIJj33iyloqSW1bcU4unlzgtPf0pQsIYfXlfIvj0NHKnu4Ee3zcXpcPLGaweYNz+R+IRA3nyjBH9/NUsWp/LJjqMYDRa+f2UOFQePc7x9iKUL46mt66VvwMQPVuZztL6PvoExli1M4YuyZpQKOXnpEXyyp45ZWZFo1Aq2761n6ZwkrBN29le18b2CeCpqO4gJ9UWrVrK3qpW52TGMjJkY0Y8TF+pHSW07hSkRGM1WxsYtBGlVNPTqmZ0eR4/OxJK5BQD0GG2EaRTUHO8lQqOkf8yEY1wHQN/IKAq5jJaBEe5dUszyjIQzuq7OBk6nk97eXioqKvDz8yM7O/ucjcnd3d3x9/cnJiaGzMxMioqKyM7OJjAwEJvNxvHjxykrK6O0tJTa2lo6OjoYHR0VAySLxUJ1dbWYUnCZzriiYblcjru7OwqFAnd3d9zd3XE4HDz11FN0dHTg4eGBu7v7jL1Murq6pimZwsPD6erqOu0+MplMTHmcT1w0pHuqJ5FrtlJTUxN+fn7MmjVrRubIp8KZkq5Op6OiooK+vj5ycnLE4XlRUVH4+/uj0+mora3lwIEDdA+288MHvse9L94IwO+ue45n73+Tvu5+Dh48yLjFwIMb7uTeZ26ivb6H+1c8RfXeegC8fb148JXbuPxH89jyt7384c6NWExW3D3k3PvH61h+02w+3LCXl9a/j8PhID4jnN++8mPM41bW/2gDI/16AkJ8+O1LN4PTyWN3vIpu2EhyTiR3PfZ96qo6ePE/tyCRwE8fvoLAcA3/u+49pE4Pbr1rJQsvy+D/NnzBsaoOZs1LYNlVOWx+q4ym2j5WrZ5NblEsuzY3ExeTyj1rr8RL5cEbL5XQ29fBnEVhHKvp4v/+vpvi+eH4+at45cU9BAZ6s+KKLHZ9dozurhFuvLGY/v4xdn52jGt+mI/TCe9/cIgVl2agVMjZ8Led
```

Comment:
Maybe remove as this one is not the vector-function you are differentiating?

Reply:
removed image